src/components/DigitalAssetsGrid/digitalAssetsGrid.scss

```diff
@@ -3,34 +3,21 @@
 
 .digital-assets-grid {
   all: initial;
+  display: block;
   position: relative;
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  grid-gap: 20px;
+  width: 100%;
+  max-width: 1040px;
+  font-size: 0;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: -20px;
 
-  &.-loading {
-    @include flex-row-center;
-
-    height: 100%;
-    width: 100%;
-  }
-
-  &.-blue {
-    padding: 0 20px 30px 20px;
-
-    &::after {
-      display: block;
-      position: absolute;
-      right: 20px;
-      bottom: 0;
-      left: 20px;
-      height: 1px;
-      background: rgba($white, 0.15);
-      content: '';
-    }
-  }
-
-  @media only screen and (max-width: 1400px) {
-    grid-template-columns: 1fr 1fr;
+  > .box {
+    width: 33.333%;
```

Comment:
we have fixed width for the digital asset item 320px

Reply:
This is not true. Cards are resized to fit the container. This can be seen in the task task-11-adaptation file